postgres/tests/common.py

```diff
@@ -11,6 +14,14 @@
 PASSWORD = 'datadog'
 DB_NAME = 'datadog_test'
 POSTGRES_VERSION = os.environ.get('POSTGRES_VERSION', None)
+
+if POSTGRES_VERSION is not None:
+    USING_LATEST = POSTGRES_VERSION.endswith('latest')
+else:
+    USING_LATEST = False
+
+if USING_LATEST is True:
+    POSTGRES_VERSION = str(maxsize)
```

Comment:
What does this do?

Reply:
This is to preserve the case where the POSTGRES_VERSION can be None as it's a case covered and tested against: https://github.com/DataDog/integrations-core/blob/aee18148cebf5026099abde7bc218d3ba8d2e75c/postgres/tests/utils.py#L9 
Then create an env var USING_LATEST for the requires_static_version test marker